extensions/arc/package.json

```diff
@@ -96,7 +96,7 @@
       "view/item/context": [
         {
           "command": "arc.openDashboard",
-          "when": "view == azureArc && viewItem != postgresInstances",
+          "when": "view == azureArc",
```

Comment:
I don't think we should enable the postgres dashboard.  None of it works yet.

Reply:
Ah, sorry. I thought the work Naia did made it at least somewhat usable. I'll revert this then.